specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/stable/2023-03-01/appconfiguration.json

```diff
@@ -821,7 +821,7 @@
         "tags": [
           "KeyValues"
         ],
-        "description": "Gets the properties of the specified key-value.",
+        "description": "Gets the properties of the specified key-value. NOTE: This operation is intended for use in ARM Template deployments. For all other scenarios involving App Configuration key-values the data plane API should be used instead.",
```

Comment:
Just want to get your thoughts... How do you feel about mentioning Bicep/Terraform besides of ARM template?

Reply:
my thought is to stick with the most fundamental piece. Kind of how SDK team refers to their JavaScript (and TypeScript) SDK as the JS SDK.

Reply:
Sounds good.